datadog_checks_dev/datadog_checks/dev/tooling/commands/meta/scripts/github_user.py

```diff
@@ -0,0 +1,34 @@
+# (C) Datadog, Inc. 2020-present
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+
+import click
+import requests
+
+from ...console import CONTEXT_SETTINGS, abort, echo_success
+
+
+@click.command(context_settings=CONTEXT_SETTINGS, short_help='Lookup Github username by email.')
+@click.argument('email')
+def email2ghuser(email):
```

Comment:
Does it make sense for this to be a cli function rather that just an utility method? (Thinking it would be called by a script that fixes codeowners)

Reply:
We thought it might not make sense to try to call the API every time the script runs, and instead just use it for the initial conversions and recommend people put their username in the codeowners file from now on.